tools-edge/oas-checker/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:14.11.0-alpine3.12
+
+RUN apk update && \
+    apk upgrade && \
+    npm install -g npm install -g @stoplight/spectral
+
```

Comment:
There is a typo in the command that installs spectral 'npm install -g' is duplicated. Although the image is built correctly, please fix that as it is installing two aditional npm packages (npm itself and install) I think it's not the intended.

Reply:
:D fixed